File: PHP/fizzbuzz_oneliner.php
<?php

// FizzBuzz One liner Using PHP 7.4 Arrow functions & nested ternary operators
// Author: @Scarwolf

foreach (array_map(fn($v) => !($v % 15) ? "FizzBuzz" : (!($v % 5) ? "Buzz" : (!($v % 3) ? "Fizz" : $v)), range(1, 100)) as $value) { echo $value . '<br>'; }

File: Swift/FizzBuzzOneliner.swift
// FizzBuzz oneliner using nested ternary operator
// Author: @sedlakp
(1...100).forEach { ($0 % (3*5) == 0) ? print("FizzBuzz") : ($0 % 3 == 0) ? print("Fizz") : ($0 % 5 == 0) ? print("Buzz") : print($0) }